rsts/getting_started/run_on_flyte.rst

```diff
@@ -25,50 +34,57 @@ Make Sure you have followed :ref:`getting-started-firstrun` and have a running d
 .. _getting-started-run-flyte-laptop:
 
 ****************************
-Running your Flyte Workflows
+Running Your Flyte Workflows
 ****************************
 
-Registration
-============
-Registration is the process of shipping your code to Flyte backend. This creates an immutable, versioned record of your code with FlyteAdmin service.
-
-Register your workflows
------------------------
+Register Your Workflows
+=======================
+Registration is the process of shipping your code to the Flyte backend. This creates an immutable, versioned record of your code with the FlyteAdmin service.
 
-From within root directory of ``flyteexamples`` you created :ref:`previously <getting-started-first-example>`
-feel free to make any changes and then register ::
+From within root directory of ``flyteexamples`` you previously created with the :ref:`Write Your First Workflow tutorial <getting-started-first-example>`,
```

Comment:
The* root directory?

Reply:
Replacing the word "previously" which was a link to the getting started example by "Write Your First Workflow tutorial"